tests/UnitTests/GT.Core.Tests/Services/GTLocationServiceTests.cs

```diff
@@ -53,5 +57,135 @@ public async Task AddAsync_AddValidNewLocation_Succeeds(string? inputLocationNam
 				.Be(callbackResult.Name).And
 				.Be(dto.Name);
 		}
+
+		[Theory]
+		[InlineData(null, "id which should be replaced by service")]
+		[InlineData("", null)]
+		public async Task AddAsync_AddInValidNewLocation_FailsAndReturnsNull(string? inputLocationName, string? inputTempId)
```

Comment:
Proposal: AddInvalidNewLocation (invalid is one word - review upper case).

Reply:
Agreed, will fix in commit.